tpl/bootstrap.php

```diff
@@ -490,7 +490,7 @@
 <?php
 endif;
 ?>
-				<div id="status" role="alert" class="alert alert-info<?php echo empty($STATUS) ? ' hidden' : '' ?>">
+				<div id="status" role="alert" class="alert alert-<?php echo $STATUS === 'Paste was properly deleted.' ? 'success' : 'info' ?><?php echo empty($STATUS) ? ' hidden' : '' ?>">
```

Comment:
I wouldn't change the `#status` box design for this case, as it will then stay in `alert-success` (green) mode even when other informational (blue) messages get displayed.

Reply:
I believe that the check `$STATUS === 'Paste was properly deleted.' ? 'success' : 'info'` will ensure that the alert is green only when it is the paste message being displayed? Otherwise it will still be the original blue alert.

Reply:
Granted, right, it will work because you force a reload with the timeout.